src/ui/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,32 @@
+import { InputHTMLAttributes, ReactElement } from "react";
+import Select from "react-select";
+import classNames from "classnames";
+import { IconOption } from "./IconOption";
+import { Option } from "@src/types";
+import "./dropdown.scss";
+
+export type DropdownProps = {
+  label?: string;
+  errorMessage?: string;
+  options: readonly Option[];
+} & InputHTMLAttributes<HTMLSelectElement>;
+
+export function Dropdown(props: DropdownProps): ReactElement {
```

Comment:
I'd use JSX.Element because it's in global scope and no need to import ReactElement all the time.